docs/development/AzureOAIDeployment/DeployingAzureOAI.md

```diff
@@ -0,0 +1,148 @@
+OpenAI deployment. Due to its additional features and manageability, we recommend using the Azure
+OpenAI Service. This document provides easy step-by-step instructions on how to deploy the Azure
+OpenAI Service using Bicep files.
+
+There are two things needed to begin a chat experience with Azure OpenAI Service, an Azure OpenAI
+Service account and an Azure OpenAI Deployment. The Azure OpenAI Service account is a resource that
+contains multiple different model deployments. The Azure OpenAI Deployment is a model deployment
+that can be called via an API to generate responses.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- An active Azure subscription
+- Azure CLI or Azure PowerShell installed
+- Proper permissions to create resources in your Azure subscription
+
+## Steps to Deploy
+
+### 1. Getting and modifying the Bicep file
+
+Clone the repository and navigate to the `./docs/development/AzureOAIDeployment` directory:
+
+```sh
+git clone www.github.com/PowerShell/AIShell
+cd AIShell/docs/development/AzureOAIDeployment
+```
+
+You will need to modify the `./main.bicep` file to include your own values. You will have to modify
+the parameters at the top of the file.
+
+```bicep
+@description('This is the name of your AI Service Account')
+param aiserviceaccountname string = '<Insert own account name>'
+
+@description('Custom domain name for the endpoint')
+param customDomainName string = '<Insert own unique domain name>'
+
+@description('Name of the deployment')
+param modeldeploymentname string = '<Insert own deployment name>'
+
+@description('The model being deployed')
+param model string = 'gpt-4'
+
+@description('Version of the model being deployed')
+param modelversion string = 'turbo-2024-04-09'
+
+@description('Capacity for specific model used')
+param capacity int = 80
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@allowed([
+  'S0'
+])
+param sku string = 'S0'
+```
+
+The above is defaulted to use your resource groups location as the location for the account and
+`gpt-4` version `turbo-2024-04-09`. You can modify this based on the particular model you feel best
+fits your needs. You can find more information on available models at
+[Azure OpenAI Service models][03]. Additionally, you may need to modify the capacity of the
+deployment based on what model you use, you can find more information at
+[Azure OpenAI Service quotas and limits][04].
+
+### 2. Deploy the Azure OpenAI Service
+
+Now that you have modified the bicep files parameters, you are ready to deploy your own Azure OpenAI
+instance! Simply use either Azure CLI or Azure PowerShell to deploy the bicep files.
+
+#### Using Azure CLI
+
+```sh
+az deployment group create --resource-group <resource group name> --template-file ./main.bicep 
+```
+
+#### Using Azure PowerShell
+
+```powershell
+New-AzResourceGroupDeployment -ResourceGroupName <resource group name> -TemplateFile ./main.bicep
+```
```

Comment:
The sections below simply list the AzPS and AzCLI commands in their own code block, maybe do that for this section too and get rid of the `Using Azure CLI/Using Azure PowerShell` sub-sections?